### DISCUSSION TITLE
Mermaid構文でsequenceDiagramを記述する際、横長になると図の上下に余分な空白が入ってしまう。

### DISCUSSION THREAD

**Discussion by IoriGunji** [increments/qiita-discussions#481] [要望・アイデア・不具合]
Mermaidの実装、ありがとうございます。
使ってみたところ、意外と使いやすくて今後も使っていきたいと思っています。

一点ご相談なのですが、表題の通り、図が横長になってしまうと、図の上下に余計な空白が入ってしまいます。
SVGの `hight` 調整はなかなかに厄介だとは思いますが、ご対応いただければ幸いです。
ご検討のほど、よろしくお願いいたします。

以下、参考画像と事象が起こっているページになります。

![rapture_20230529175335](https://github.com/increments/qiita-discussions/assets/58961066/2b7b3a12-dbaa-4640-96b0-3c897140038b)

https://qiita.com/IoriGunji/items/de9ccbeda4e7682c711b

**Comment by tanaka-asami415** [comment]
@IoriGunji 
ご報告ありがとうございます🙏
こちら調査したところ、すぐに修正できる内容だったため修正を進めております！
もうしばらくお待ちいただけると幸いです🙇

**Comment by wataru86** [comment]
@IoriGunji
お待たせしました🙇
こちら修正が完了いたしました！

![image](https://github.com/increments/qiita-discussions/assets/34445284/69a3bb6b-c81d-4769-aca0-47dc28d9f852)

**Reply by yarakos95** [reply]
横から失礼します 🙇 

問題提起されている余分な空白は解消されているようですが、ダイアグラムの下部が見切れてしまっているようです。

![image](https://github.com/increments/qiita-discussions/assets/75878492/e1d10ca7-2b6b-4b56-84a5-ed3f3f910faf)

**Reply by wataru86** [reply]
@Yarakashi-Kikohshi

ご報告ありがとうございます🙏 修正しました！

@IoriGunji

上記の見切れてしまう問題も含め、修正が完了しました！

![image](https://github.com/increments/qiita-discussions/assets/34445284/ed289c35-3d2c-40e9-9ef8-b17b76c48943)

**Reply by IoriGunji** [reply]
@wataru86 
ご対応ありがとうございます！
